test/k8sT/Services.go

```diff
@@ -1327,7 +1327,9 @@ var _ = Describe("K8sServicesTest", func() {
 
 				Context("Tests with vxlan", func() {
 					BeforeAll(func() {
-						DeployCiliumAndDNS(kubectl, ciliumFilename)
+						DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
```

Comment:
Maybe we should create a separate Context within the vxlan one, and deploy Cilium with the host-fw only there? Asking, as enabling the host-fw changes a packet path, and not all users will enable it.

Reply:
Yep, makes sense. I'll do that.